### DISCUSSION TITLE
Readme error: .NET Desktop Runtime needed, not .NET Runtime

### DISCUSSION THREAD

**Discussion by QuestioningEspecialy** [Tyrrrz/DiscordChatExporter#1208] [General]
On launch, I was confronted with the install prompt for ".NET (Desktop) Runtime" (pretty sure it had parenthesis in there), so the program needs the _Desktop_ Runtime installed even though the [Readme ](https://github.com/Tyrrrz/DiscordChatExporter/blob/master/Readme.md)says to just install the .NET Runtime.

Windows 10 64bit
v2.42.8

**Comment by Tyrrrz** [comment]
The Desktop Runtime is only required on Windows. The readme note you're referencing only provides download links for macOS and Linux.

**Reply by QuestioningEspecialy** [reply]
Makes sense, but a simple clarification would be to change the wording to this:
> On Windows, the Desktop Runtime should be installed automatically
